tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts

```diff
@@ -129,33 +125,34 @@ function getRouter(options: PfeDevServerInternalConfig) {
         }
       })
 
-  // Redirect `components/jazz-hands/*` to `components/pf-jazz-hands/*` for requests not previously handled
+      // Redirect `components/jazz-hands/*` to `components/pf-jazz-hands/*` for requests not previously handled
       .get(`/${componentSubpath}/:element/:splatPath*`, async (ctx, next) => {
         const { element, splatPath } = ctx.params;
         const prefixedElement = deslugify(element);
+        // if we are passing a folder ie (/elements/pf-jazz-hands/demo/) redirect to the elements dir with splat.
         if (await isDir(new URL(`/${elementsDir}/${prefixedElement}`, import.meta.url).href)) {
           ctx.redirect(`/${elementsDir}/${prefixedElement}/${splatPath}`);
+        }
+        // If the splat exits and is a lightdom.css | lightdom-shim.css file redirect to the splat path
+        if (splatPath && splatPath.includes('lightdom')) {
+          ctx.redirect(`/${elementsDir}/${prefixedElement}/${splatPath}`);
         } else {
           return next();
         }
       })
 
+      // TODO: Discovery to see if this redirect is still needed.
+      // Redirect `components/jazz-hands/*.js` to `elements/pf-jazz-hands/*.ts`
+      .get(`/${componentSubpath}/:element/:splatPath*.js`, async ctx => {
+        const { element, splatPath } = ctx.params;
+        const prefixedElement = deslugify(element);
+        ctx.redirect(`/${elementsDir}/${prefixedElement}/${splatPath}.ts`);
+      })
+
+      // SET ETAG
       .get(`/${elementsDir}/:element/:splatPath*.(css|js)`, async function(ctx, next) {
         ctx.response.etag = performance.now().toString();
         return next();
-      })
-
-  // Redirect `core/pfe-core/controllers/thingy.js` to `core/pfe-core/controllers/thingy.ts` for requests not previously handled
-      .get(`/core/pfe-core/:splatPath*.js`, async ctx => {
-        ctx.redirect(`/core/pfe-core/${ctx.params.splatPath}.ts`);
-      })
-
-  // Redirect `elements/pf-jazz-hands/*.js` to `elements/pf-jazz-hands/*.ts` for requests not previously handled
-      .get(`/${elementsDir}/:element/:splatPath*.js`, async ctx => {
-        const { element, splatPath } = ctx.params;
-        if (element.startsWith(tagPrefix) && !splatPath.includes('/')) {
-          ctx.redirect(`/${elementsDir}/${element}/${splatPath}.ts`);
-        }
```

Comment:
found this block to be unnecessary given the `liveReloadTsChangesMiddleware` that was added, the attempt was the same, but i think the middleware might be a better place for this particular redirect.